validation/glean/baseline.1.all.pass.json

```diff
@@ -25,33 +25,82 @@
     "boolean": {
       "examples.boolean_example": true
     },
+    "labeled_boolean": {
+      "examples.labeled_boolean_example": {
+        "a": true,
+        "b": false
+      }
+    },
     "string": {
       "examples.subcategory.string_example": "\u82f1\u6f22\u5b57\u5178"
     },
+    "labeled_string": {
+      "examples.subcategory.labeled_string_example": {
+        "longer_key_1": "ABCD",
+        "an_even_longer_key_2": "EFGH"
+      }
+    }
```

Comment:
Looks like you're missing a comma here

Reply:
Weird.  I guess my passing tests aren't actually running this, then.  It would have caught these syntax errors.